pkg/linksharing/web/single-object.html

```diff
@@ -54,9 +53,10 @@ <h3 class="mb-3">You’re getting this file from all over the world.</h3>
           </div>
         </div>
       </div>
+      <p><img class="collapser" src="{{.Base}}/static/img/collapse.svg" onclick="toggleSideBar()" alt="collapse icon"></p>
     </div>
-
   </div>
+  <p><img class="expander" src="{{.Base}}/static/img/expand.svg" onclick="toggleSideBar()" alt="collapse icon"></p>
```

Comment:
Is alt here correct?

```suggestion
  <p><img class="expander" src="{{.Base}}/static/img/expand.svg" onclick="toggleSideBar()" alt="expand icon"></p>
```